$NetBSD$

This file would normally be created by the configure script, but we
don't run it.

--- _notmuch_config.py.orig	2024-03-01 22:02:43.485120189 +0000
+++ _notmuch_config.py
@@ -0,0 +1,3 @@
+NOTMUCH_VERSION_FILE='../../version.txt'
+NOTMUCH_INCLUDE_DIR='../../lib'
+NOTMUCH_LIB_DIR='../../lib'
